# HG changeset patch
# User Cole Robinson <crobinso@redhat.com>
# Date 1260376118 18000
# Node ID b69f023023414ab3ca7d7f818beb710a04aa409b
# Parent  79c021ce411081dbb06b90952a5cf0003b170043
manager: Change the selection to the row we right click on

diff -r 79c021ce4110 -r b69f02302341 src/virtManager/manager.py
--- a/src/virtManager/manager.py	Wed Dec 09 10:58:47 2009 -0500
+++ b/src/virtManager/manager.py	Wed Dec 09 11:28:38 2009 -0500
@@ -1049,7 +1049,7 @@
         _iter = model.get_iter(path)
 
         self.popup_vm_menu(model, _iter, event)
-        return True
+        return False
 
     def popup_vm_menu(self, model, _iter, event):
         if model.iter_parent(_iter) != None:
@@ -1090,8 +1090,6 @@
 
             self.connmenu.popup(None, None, None, 0, event.time)
 
-        return False
-
 
     #################
     # Stats methods #
